packages/core/src/constants/chainId.ts

```diff
@@ -20,11 +22,11 @@ export enum ChainId {
 }
 
 export const CHAIN_NAMES = {
-  [ChainId.Mainnet]: 'Mainnet',
-  [ChainId.Ropsten]: 'Ropsten',
-  [ChainId.Kovan]: 'Kovan',
-  [ChainId.Rinkeby]: 'Rinkeby',
-  [ChainId.Goerli]: 'Goerli',
+  [Mainnet.chainId]: Mainnet.chainName,
+  [Ropsten.chainId]: Ropsten.chainName,
+  [Rinkeby.chainId]: Rinkeby.chainName,
+  [Goerli.chainId]: Goerli.chainName,
+  [Kovan.chainId]: Kovan.chainName,
   [ChainId.BSC]: 'BSC',
```

Comment:
in future phase we can use access chains object via index instead of define CHAIN_NAMES, MULTICALL_ADDRESSES...